package.json

```diff
@@ -59,6 +59,7 @@
     "node": ">=10"
   },
   "devDependencies": {
+    "aedes": "^0.44.2",
```

Comment:
You can install only types by using `npm install --save @types/aedes`

Reply:
The package doesn't exist.

Reply:
Oh, ok ignore my comment so :)